mmseg/models/backbones/vit.py

```diff
@@ -344,21 +343,21 @@ def _pos_embeding(self, img, patched_img, pos_embed):
                 raise ValueError(
                     'Unexpected shape of pos_embed, got {}.'.format(
                         pos_embed.shape))
-            pos_embed = self.resize_pos_embed(pos_embed, img.shape[2:],
-                                              (pos_h, pos_w), self.patch_size,
+            pos_embed = self.resize_pos_embed(pos_embed, hw_shape,
+                                              (pos_h, pos_w),
                                               self.interpolate_mode)
         return self.drop_after_pos(patched_img + pos_embed)
 
     @staticmethod
-    def resize_pos_embed(pos_embed, input_shpae, pos_shape, patch_size, mode):
+    def resize_pos_embed(pos_embed, input_shpae, pos_shape, mode):
         """Resize pos_embed weights.
 
         Resize pos_embed using bicubic interpolate method.
         Args:
             pos_embed (torch.Tensor): pos_embed weights.
             input_shpae (tuple): Tuple for (input_h, intput_w).
             pos_shape (tuple): Tuple for (pos_h, pos_w).
-            patch_size (int): Patch size.
+            mode (str): Algorithm used for upsampling.
```

Comment:
It would be better to descript the `Args` in more detail, and we don’t need abbreviations in the description, e.g. pos_embed should be position embedding.